### DISCUSSION TITLE
SharePoint Add-In retirement

### DISCUSSION THREAD

**Discussion by Heavm00** [microsoft-search/pnp-modern-search#3958] [Q&A] [answered]
I recently read about the SharePoint Add-In retirement in Microsoft 365 and  announcemant of using a tenant app catalog will stay supported and enables 3rd parties to offer their SharePoint Add-Ins until April 2nd, 2026 (https://learn.microsoft.com/de-de/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins).

How does the announcement influences the development of Search Webpart v4? Does this mean that the web parts will no longer work from April 2026?

**Comment by kasperbolarsen** [comment, marked as answer]
Happy you asked, as it proves that this question belongs in the Q&A section: https://microsoft-search.github.io/pnp-modern-search/QnA/
